Sources/peertalk/include/PTChannel.h

```diff
@@ -7,9 +7,9 @@
 #import <netinet/in.h>
 #import <sys/socket.h>
 
-#import <peertalk/PTProtocol.h>
-#import <peertalk/PTUSBHub.h>
-#import <peertalk/PTDefines.h>
+#import "PTProtocol.h"
```

Comment:
This doesn't seem right to me. Xcode already allows you to `Always Search User Header Paths` (I forget if that's the exact setting name), and it can find these  headers fine with the original `<>` form.